__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+//SETUP AND SHUTDOWN
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+// TEST SUITE
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("200: Should recieve a 200 code for a sucessful request", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+
+    test('200: Should respond with a body of topic objects in an array, with "slug" and "description properties"', () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const topicsArr = body.topics;
+
+          expect(Array.isArray(topicsArr)).toBe(true);
+          topicsArr.forEach((topic) => {
+            expect(topic).toHaveProperty("slug");
+            expect(topic).toHaveProperty("description");
+          });
+        });
+    });
+
+    test("500: Should respond with a 500 code if DB is not connected or seeded ", () => {
```

Comment:
This test isn't necessary or necessarily correct, the API shouldn't run at all without the DB connection. So this isn't needed. Your test file shouldn't  be querying the db directly, db interaction should be done in models.